pennylane/qchem/basis_data.py

```diff
@@ -20,18 +20,28 @@
 
 import itertools
 
+# fmt: off
 atomic_numbers = {
-    "H": 1,
-    "He": 2,
-    "Li": 3,
-    "Be": 4,
-    "B": 5,
-    "C": 6,
-    "N": 7,
-    "O": 8,
-    "F": 9,
-    "Ne": 10,
+    'H': 1, 'He': 2, # Period 1
```

Comment:
[optional] Please cite NIST or IUPAC pages/papers for this. e.g., [this](https://iupac.org/what-we-do/periodic-table-of-elements/) or [this](https://www.nist.gov/pml/periodic-table-elements).